README.md

```diff
@@ -59,6 +59,17 @@ repos:
         args: [--format=pdf]
 ```
 
+If you want the converted files in a separate folder
+
+```yaml
+repos:
+-   repo: https://github.com/fernandezpablo85/notebook_convert
+    rev: v0.0.1
+    hooks:
+    -   id: notebook_convert
+        args: [--format=rst, --fileDestinationMode=mirror_folder]
```

Comment:
I'd rather have a shorter name like `--output_dir`

Reply:
but, that sounds like you could put a path on it, and that is not the case. But maybe it must be. Do you prefer that I change the behavior to allow set a relative dir ??